decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDeciderCli.java

```diff
@@ -68,6 +69,8 @@
     private final OptionSpec<String> beforeDateOpt;
     private final OptionSpec<String> includeInstrumentFilterOpt;
     private final OptionSpec<String> excludeInstrumentFilterOpt;
+    private final OptionSpec<String> overrideBasesMaskOpt;
+    private final OptionSpec<Integer> minAllowedEditDistance;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "minAllowedEditDistance" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)